src/components/10-atoms/fieldset/ui.test.js

```diff
@@ -67,7 +67,7 @@ test('should responsive stretch', async t => {
     ];
   });
 
-  const expected = JSON.stringify(['529px', '0px']); // 529px width of the radio correspond to 100% in a 575px window
+  const expected = JSON.stringify(['545px', '0px']); // 545px width of the radio correspond to 100% in a 575px window
```

Comment:
The last number was forgotten, maybe just replace this comment with `100% width` or something.

Reply:
You mean 575px? Thats correct. At this test the browser is resized to 575px width.